apps/modernization-ui/src/components/Table/Table.tsx

```diff
@@ -19,6 +20,7 @@ type OldSelectionHandler = (event: ChangeEvent<HTMLInputElement>, item: any) =>
 export type Cell = {
     id: string | number;
     title: ReactNode | ReactNode[] | string;
+    actions?: any;
```

Comment:
issue:  The actions are a feature of Patient Profile only and may have to be added as a column instead of being handled in the `Table` component.

Reply:
Don't forget this one